src/newfilewizard/newfilewizard.cpp

```diff
@@ -139,17 +157,21 @@ void NewFileDialog::onCreateFile()
     try
     {
         FileManager().createFile(dirName + '/' + fileName);
-    } catch (const QException&)
+    } catch (const FileOpeningFailure&)
     {
         QMessageBox::warning(this, "Error", "Unable to create file");
         return;
     }
     QMessageBox::information(this, "File Created", "Specified file has been successfully created.");
 
     // check if filename is an empty string
-    if(fileName.isEmpty())
+    if (fileName.isEmpty())
```

Comment:
mFileName  = fileName.isEmpty() ? QString() : dirName + '/' + fileName;

Reply:
fixed that